tools/SourceKit/tools/sourcekitd-test/sourcekitd-test.cpp

```diff
@@ -105,14 +105,17 @@ static unsigned resolveFromLineCol(unsigned Line, unsigned Col,
                                    llvm::MemoryBuffer *InputBuf);
 static std::pair<unsigned, unsigned>
 resolveToLineCol(unsigned Offset, StringRef Filename,
-                 const llvm::StringMap<TestOptions::VFSFile> &VFSFiles);
-static std::pair<unsigned, unsigned> resolveToLineCol(unsigned Offset,
-                                                  llvm::MemoryBuffer *InputBuf);
+                 const llvm::StringMap<TestOptions::VFSFile> &VFSFiles,
+                 bool ExitOnError = true);
+static std::pair<unsigned, unsigned>
+resolveToLineCol(unsigned Offset, llvm::MemoryBuffer *InputBuf,
+                 bool ExitOnError = true);
```

Comment:
Suggested doc comment:
```cpp
If the position could not be resolved and \p ExitOnError is \c true, the process exists with an error message. Otherwise, it returns {0, 0}.
```

Reply:
Done, also added one to the `getBufferForFilename`.